### PR TITLE
make sure new vars are copied to heroku

### PR DESCRIPTION
**Pull request by mbertrand**
#### What are the relevant tickets?
N/A

#### What's this PR do?
Make sure new vars are copied to heroku

### CODE REVIEW

File: pillar/heroku/ocw-studio.sls [hunk 1]
```diff
@@ -77,6 +77,7 @@ heroku:
     AWS_STORAGE_BUCKET_NAME: 'ol-ocw-studio-app-{{ env_data.env }}'
     CONTENT_SYNC_BACKEND: content_sync.backends.github.GithubBackend
     CONTENT_SYNC_PIPELINE: content_sync.pipelines.concourse.ConcourseGithubPipeline
+    CONCOURSE_URL: {{ env_data.CONCOURSE_URL }}
     CONCOURSE_USERNAME: oldevops
     CONCOURSE_PASSWORD: __vault__::secret-concourse/web>data>data>admin_password
```
Comment (mbertrand) [inline pillar/heroku/ocw-studio.sls:82]: This doesn't seem to be carried over to heroku, any idea why?  Is the value missing from vault?

Comment (blarghmatey) [reply pillar/heroku/ocw-studio.sls:82]: It's because the path isn't quite right. It needs to be `secret-concourse/data/web`

**Comment by blarghmatey** [comment]
:+1:

File: pillar/heroku/ocw-studio.sls (hunk 1 above)
Comment (mbertrand) [reply pillar/heroku/ocw-studio.sls]: Thanks, just changed it to ```
CONCOURSE_PASSWORD: __vault__::secret-concourse/data/web>data>data>admin_password
```

**Pull request merged**